### PR TITLE
Add HPE ProCurve 2910al-48G (J9147A)

### PR DESCRIPTION
**Pull request by basictheprogram**
Added new device-type HPE ProCurve 2910al-48G

### CODE REVIEW

File: device-types/HPE/ProCurve-2910-48G.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+u_height: 1
+comments: '[ProCurve 2910al-48G Specs](https://support.hpe.com/hpesc/public/docDisplay?docId=emr_na-c01844343)'
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 92
```
Comment (danner26) [inline device-types/HPE/ProCurve-2910-48G.yaml:17]: Where did you get this value from? According to the datasheet, the values should be `maximum_draw: 105` and `allocated_draw: 64`

The datasheet says that the max draw is 105W and the idle power is 64W

File: device-types/HPE/ProCurve-2910-48G.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+model: ProCurve 2910al-48G
+slug: hpe-procurve-2910al-48g
+part_number: J9147A
+u_height: 1
+comments: '[ProCurve 2910al-48G Specs](https://support.hpe.com/hpesc/public/docDisplay?docId=emr_na-c01844343)'
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
```
Comment (danner26) [inline device-types/HPE/ProCurve-2910-48G.yaml:14]: I dont use these so I do not know, but it appears that the J9147A has a redundant 12V DC port for backup power. If this is true, please add that as well

File: device-types/HPE/ProCurve-2910-48G.yaml [hunk 3]
```diff
@@ -0,0 +1,112 @@
+---
+manufacturer: HPE
+model: ProCurve 2910al-48G
+slug: hpe-procurve-2910al-48g
+part_number: J9147A
+u_height: 1
+comments: '[ProCurve 2910al-48G Specs](https://support.hpe.com/hpesc/public/docDisplay?docId=emr_na-c01844343)'
+is_full_depth: false
+console-ports:
```
Comment (danner26) [inline device-types/HPE/ProCurve-2910-48G.yaml:11]: Please add `airflow` information if you have it

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/HPE/ProCurve-2910-48G.yaml (hunk 2 above)
Comment (basictheprogram) [reply device-types/HPE/ProCurve-2910-48G.yaml:14]: Was just sent a picture of the device, asking about the 12V DC, I'll update the PR when I get confirmation.

File: device-types/HPE/ProCurve-2910-48G.yaml (hunk 3 above)
Comment (basictheprogram) [reply device-types/HPE/ProCurve-2910-48G.yaml:11]: Do not have it, but tracking down the information. I'll update the PR when I get confirmation.

Comment (danner26) [reply device-types/HPE/ProCurve-2910-48G.yaml:11]: Thanks, I will mark this as a draft. Once it is ready please mark it for review and ping me!

**Pull request closed without merging**